msvc-build-launcher.cmd

```diff
@@ -35,5 +35,15 @@ if "%ERRORLEVEL%"=="0" (
   echo Windows SDK 6.1 not found to build Windows 64-bit version
 )
 
+REM now for arm 64-bit
+REM Cross compile for arm64
+call VCVARSx86_arm64
+if "%ERRORLEVEL%"=="0" (
+  cl /D "GUI=0" /D "WIN32_LEAN_AND_MEAN" launcher.c /O2 /link /MACHINE:arm64 /SUBSYSTEM:CONSOLE /out:setuptools/cli-arm64.exe
+  cl /D "GUI=1" /D "WIN32_LEAN_AND_MEAN" launcher.c /O2 /link /MACHINE:arm64 /SUBSYSTEM:WINDOWS /out:setuptools/gui-arm64.exe
+) else (
+  echo Windows SDK 6.1 not found to build Windows 64-bit version
+)
+

```

Comment:
I'd prefer if I could build the launcher. Do you know how I could acquire a Windows ARM machine (perhaps through Azure) that would build the executable? Did you have to do anything special to get the build to run?

Reply:
I think executables can be cross-compiled from an x64 machine as well. But for execution, you need to get one of the windows on arm machines. I have tested on a surface pro x machine. I don't think we have any machines available yet azure yet. 

The launcher works out of the box, I just had to compile it with the right target machine.